topicExtractor.py

```diff
@@ -135,14 +137,34 @@ def fitTopicModel(self):
         """
         docs = self.videoData.combinedTranscript["Combined Lines"].tolist()
         timestamps = self.videoData.combinedTranscript["Start"].tolist()
+        vectorizerModel = (
+            getVectorizer(self.videoData.combinedTranscript) if useKeyBERT else None
+        )
 
         callAttemptCount = 0
         while callAttemptCount < self.callMaxLimit:
             try:
-                topics, probs = self.topicModel.fit_transform(docs)
+                self.initializeTopicModel(vectorizerModel)
+                self.topics, probs = self.topicModel.fit_transform(docs)
```

Comment:
I feel the `probs` variables results should be logged since it is returned and not used anywhere else

Reply:
By logging probs, do you mean I assign it to the class for the topic modeler?
`self.topics, self.probs = self.topicModel.fit_transform(docs)`
I can do that, I just want to make sure that is what you meant here. You are right in stating it is not used elsewhere again, I do believe that we never actually need it for anything else.

Reply:
>By logging probs, do you mean I assign it to the class for the topic modeler?
`logging.info`

the tuple seems to be giving topics and probabilities. Usually we do something about the returned response . I just felt the info could be useful in logs when running the script. Again I just want the info printed to be simple

Reply:
Oh, I see. In my latest commit, I added two logging statements that mention that the fitting operation was successful. 
The `topic` and `probs` variables are not logged directly as they are just a list of numbers that do not give any information that a human could parse meaningfully. I can add a log with a small snippet of the first five elements in each of those returned lists if all we want to show is the nature of the returned data.

Reply:
If it's not human readable then forget it!